src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -1148,6 +1191,8 @@ def forward(
         cross_attn_head_mask: Optional[torch.Tensor] = None,
         encoder_outputs: Optional[Tuple[torch.FloatTensor]] = None,
         past_key_values: Optional[Tuple[torch.FloatTensor]] = None,
+        inputs_embeds: Optional[torch.Tensor] = None,
+        decoder_inputs_embeds: Optional[torch.Tensor] = None,
```

Comment:
These two arguments should be added to the docstring.

Reply:
Fixed in [d3800fd](https://github.com/huggingface/transformers/pull/20825/commits/d3800fdee2cbe30440028ca0932fe1529f4b895b)